### DISCUSSION TITLE
Static position for certain applications?

### DISCUSSION THREAD

**Discussion by RobLW** [paperwm/PaperWM#842] [Q&A] [answered]
Hi there,
Recently installed PaperWM for the first time. One thing I'm wondering is whether it's possible to have my browser of choice, always be stuck on Workspace 1 in the left most position and my email client to be the 2nd app. So if I restart those two apps for example they will auto position themselves into the same spot every time.

I saw the Winprops options but didn't look like it was possible there.
Thanks.

**Comment by Lythenas** [comment, marked as answer]
With our current winprops it is not possible.

But if I remember correctly in the past it was at least possible to open an app on a specific space, but I could be misremembering here. In any case I think this would be good to add.

Iif we finish up the dbus interface #774, I think users would be able to implement basically arbitrary rules, by watching for events and moving the window then. But that might be a bit complicated to use for just always "open firefox on space 1".

**Comment by jtaala** [comment]
Similar to the last discussion I commented on (by Rob) - yes, this could be (fairly easily) added as a `winprops`.  Let's create an issue for this one as an enhancement.
